src/main/resources/templates/fragments/header.html

```diff
@@ -0,0 +1,22 @@
+<html
```

Comment:
there should be different headers for
* not signed in
* signed in as customer
* signed in as staff 

as they all have different locations they can access and different pages they can open

Reply:
done